### ISSUE TITLE
telegraf: Plugin does not work in 1.4.x versions

### ISSUE THREAD

**Issue by windkit** [leo-project/leofs#869]
### Description
The attempt to fix issue related to url.Parse from Go 1.8 breaks the plugin
as snmpwalk expect a host name, inserting `udp://` scheme does not work

```
$ snmpwalk -v2c -cpublic udp://192.168.100.35:4000 .1.3.6.1.4.1.35450
getaddrinfo: //192.168.100.35 Name or service not known
snmpwalk: Unknown host (udp://192.168.100.35:4000)
```

```
$ snmpwalk -v2c -cpublic udp:192.168.100.35:4000 .1.3.6.1.4.1.35450
iso.3.6.1.4.1.35450.34.1.0 = STRING: "G0@192.168.100.35"
iso.3.6.1.4.1.35450.34.2.0 = Gauge32: 538
iso.3.6.1.4.1.35450.34.3.0 = Gauge32: 72424809
iso.3.6.1.4.1.35450.34.4.0 = Gauge32: 51886439
iso.3.6.1.4.1.35450.34.5.0 = Gauge32: 20537891
iso.3.6.1.4.1.35450.34.6.0 = Gauge32: 6355767
iso.3.6.1.4.1.35450.34.7.0 = Gauge32: 538
iso.3.6.1.4.1.35450.34.8.0 = Gauge32: 72504578
iso.3.6.1.4.1.35450.34.9.0 = Gauge32: 51881218
iso.3.6.1.4.1.35450.34.10.0 = Gauge32: 20609542
iso.3.6.1.4.1.35450.34.11.0 = Gauge32: 6356338
iso.3.6.1.4.1.35450.34.12.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.13.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.14.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.15.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.16.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.17.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.18.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.19.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.20.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.21.0 = Gauge32: 0
iso.3.6.1.4.1.35450.34.31.0 = Gauge32: 63
iso.3.6.1.4.1.35450.34.32.0 = Gauge32: 114881776
iso.3.6.1.4.1.35450.34.33.0 = Gauge32: 63
iso.3.6.1.4.1.35450.34.34.0 = Gauge32: 114106949

```

### Related to Commit
https://github.com/influxdata/telegraf/commit/b277e6e2d789716ff679cd8c70bdc0a39874c1dd

**Comment by windkit** [comment]
Merged and will be included from 1.4.3
https://github.com/influxdata/telegraf/pull/3328

**Comment by windkit** [comment]
Fixed in 1.4.3, released on 2017-10-25
